app/web/features/landing/HeroSection.tsx

```diff
@@ -0,0 +1,122 @@
+import { signupRoute } from "routes";
+import { service } from "service";
+import isGrpcError from "service/utils/isGrpcError";
+import stringOrFirstString from "utils/stringOrFirstString";
+
+import { useAuthContext } from "../auth/AuthProvider";
+import SignupFormContent from "../auth/signup/SignupFormContent";
+import mapAnimation from "./map-animation.json";
+
+const StyledContent = styled("div")(({ theme }) => ({
+  display: "flex",
+  flexDirection: "row",
+  width: "100%",
+  justifyContent: "space-between",
+  alignItems: "center",
+  marginBottom: theme.spacing(2),
+
+  [theme.breakpoints.down("md")]: {
+    flexDirection: "column",
+    padding: theme.spacing(8, 0, 0, 0),
+  },
+}));
+
+const StyledMobileEmbed = styled("div")(({ theme }) => ({
+  margin: theme.spacing(3),
+}));
+
+const StyledMapWrapper = styled("div")(({ theme }) => ({
+  padding: theme.spacing(2),
+  width: "55%",
+
+  [theme.breakpoints.down("md")]: {
+    width: "100%",
+    marginTop: theme.spacing(2),
+  },
+}));
+
+export default function HeroSection() {
+  const { t } = useTranslation([AUTH, GLOBAL]);
+  const router = useRouter();
+
+  const { authState, authActions } = useAuthContext();
+  const error = authState.error;
+
+  const [loading, setLoading] = useState(false);
+
+  const urlToken = stringOrFirstString(router.query.token);
+
+  const isNativeEmbed = useIsNativeEmbed();
+
+  useEffect(() => {
+    authActions.clearError();
+  }, [authActions]);
+
+  useEffect(() => {
+    if (authState.error) window.scroll({ top: 0, behavior: "smooth" });
+  }, [authState.error]);
+
+  useEffect(() => {
+    (async () => {
+      if (urlToken) {
+        setLoading(true);
+        try {
+          authActions.updateSignupState(
+            await service.auth.signupFlowEmailToken(urlToken),
+          );
+        } catch (err) {
+          Sentry.captureException(err, {
+            tags: {
+              component: "auth/signup/Signup",
+            },
+          });
+          authActions.authError(
+            isGrpcError(err) ? err.message : t("global:error.fatal_message"),
+          );
+          router.push(signupRoute);
+          return;
+        }
+        setLoading(false);
+      }
+    })();
+    // next-router-mock router isn't memoized, so putting router in the dependencies
+    // causes infinite looping in tests
+  }, [urlToken, authActions, t]); // eslint-disable-line react-hooks/exhaustive-deps
+
+  if (isNativeEmbed) {
+    return (
+      <StyledMobileEmbed>
+        {error && (
+          <Alert severity="error" sx={{ width: "100%" }}>
+            {error}
+          </Alert>
+        )}
+        {loading ? <CenteredSpinner /> : <SignupFormContent />}
+      </StyledMobileEmbed>
+    );
+  }
+
+  return (
+    <>
+      <HtmlMeta title={t("global:join_us")} />
+      <StyledContent>
+        <CouchersIntroduction />
+        <StyledMapWrapper>
+          <Lottie animationData={mapAnimation} loop={true} />
```

Comment:
I put it on the CDN: https://cdn.couchers.org/img/hero/hero-animation.json

Currently the way it's set up is that the JSON is included in the JS bundle, inflating it a lot. So it's better if we can pass a URL to Lottie to load it itself and not put it in the JS bundle.

Also, is it possible to give the map a predefined width/height so that when it takes a moment to load, it would not cause reflow?